docs/install.md

```diff
@@ -149,21 +151,21 @@ On Debian or Ubuntu, you should be able to get all these with:
 
     sudo apt-get install build-essential libcap-dev xz-utils zip \
         unzip strace curl discount git python zlib1g-dev \
-        golang-go cmake strace
+        golang-go cmake strace flex bison locales
     curl https://install.meteor.com/?release=1.8.2 | sh
 
 On Fedora 27 you should be able to get them with (as root):
```

Comment:
Fedora is up to version 34 now; are we sure the seccomp flag isn't an issue on this version as well?

I'm tempted to suggest that we should test this and probably bump the version here in the docs to something somewhat recent. I won't block on this though.